howso/types.amlg

```diff
@@ -1883,5 +1888,17 @@
 				enum ["string" "list" "assoc" "number" "null" "boolean" "any"]
 				description "The string name of the type a value should be."
 			)
+		TraineeNamePath
+			(assoc
+				type "list"
+				values {ref "TraineeName"}
+				description "The list of Trainee names that describes the hierarchical path from the Trainee called to the Trainee desired."
+			)
+		TraineeName
+			(assoc
+				type "string"
```

Comment:
Is there a `min_length` for string size? Might be a good idea to limit to 256 characters for consistency.